what_to_watch/suggest/suggestion.py

```diff
@@ -1,8 +1,16 @@
 """
 Movie suggestion handling
 """
+from typing import List
+import sys
+import pandas as pd # type: ignore
 
-def suggest_movie() -> str:
+F_DIR = 'resources/imdb_data.csv'
+ARGS = sys.argv[1:]
+IMDB_DATA = pd.read_csv(F_DIR)
+
+
+def suggest_movie(vals: List[str], data: pd.DataFrame) -> str:
```

Comment:
Looks good for a start. I was gonna add in some enums for Genre, and a base data_access class to remove the dependence from `suggest_movie`.

Maybe update the args to be a bit clearer on what they are, maybe something like `(search_types, data_handler)` ?